testing/src/main/kotlin/com/airbnb/mvrx/test/MvRxTestRule.kt

```diff
@@ -32,6 +32,7 @@ class MvRxTestRule(
         RxAndroidPlugins.setMainThreadSchedulerHandler { Schedulers.trampoline() }
         if (setRxImmediateSchedulers) setRxImmediateSchedulers()
         MvRxTestOverridesProxy.forceMvRxDebug(debugMode.value)
+        MvRxTestOverridesProxy.forceUseTestObserver(true)
```

Comment:
Did you intentionally make this not configurable? You can include it in the TestRule constructor as an optional and default-true param.

Reply:
I was thinking to make it configurable, similar as for `debugMode`, but I could think on case when you will need lifecycle aware observer for unit tests. You can't use `MvRxTestRule` for automation tests because everything would happen in main thread. Maybe robolectric ?

Reply:
Actually on second look Rx immediate schedulers are also configurable, so I make this also configurable.